public/modules/rewrite/body_management/list.html

```diff
@@ -0,0 +1,30 @@
+
+
+
+<h1 class="page-header">Bodies overview</h1>
+
+<div class="row">
+	<div class="row">
+		<div class="input-group">
+			<span class="input-group-addon"><i class="fa fa-search"></i>Search</span>
+			<input type="text" class="form-control" placeholder="Type any keywords" ng-model="vm.query" />
+		</div>
+	</div>
+	<div class="row">
+		<button type="button" class="btn btn-white pull-left" ng-if="vm.permissions.create_body" ng-click="vm.showBodyModal()">Create new body</button>
+		<button type="button" class="btn btn-white disabled pull-right">Export to Excel</button>
```

Comment:
We should reconsider what to do with the export to excel button. For now this is low priority I think.